Makefile

```diff
@@ -60,24 +60,16 @@ cluster-api-dev-helper: vendor
 	CGO_ENABLED=0 go install $(GOFLAGS) sigs.k8s.io/cluster-api-provider-aws/hack/cluster-api-dev-helper
 
 # Run tests
-test: generate fmt vet
+test: generate lint
 	go test -v -tags=integration ./pkg/... ./cmd/...
 
 # Run go fmt against code.
-fmt:
-	go fmt ./pkg/... ./cmd/...
-
-# Run go vet against code.
-vet:
-	go vet ./pkg/... ./cmd/...
-
-# Run go lint.
 lint:
-	golint || go get -u golang.org/x/lint/golint
-	golint -set_exit_status ./cmd/... ./pkg/... ./clusterctl/...
+	golangci-lint > /dev/null || scripts/install-golint.sh -b $$GOPATH/bin v1.10.2
```

Comment:
`$$GOPATH` should be `$(GOPATH)` since it's defined at the top of the file and not expected to be in the user's environment.

Why are is this installing into go's bin path and not the the default `./bin`?

Reply:
copy pasta'd. fixing up

Reply:
we don't have a bin directory at present btw